added NOTICE file

SwathiVarkala opened the pull request.

angshu commented:
What is this PR for?

mohan-13 committed:
BDI-27|Mohan|Add| Bahmni Index HTML page to proxy image

mohan-13 committed:
BDI-27|Mohan|Add| System date script to cgi-bin directory

mohan-13 committed:
BDI-27|Mohan|Add| Docker build script for bahmni proxy

mohan-13 committed:
BDI-27|Mohan|Add| Proxy Service in docker compose

mohan-13 committed:
BDI-27|Mohan|Fix| Comment volumes section in bahmni-web as defaults are commented

mohan-13 committed:
BDI-27|Mohan|Add| Docs for bahmni proxy docker package resources

mohan-13 committed:
BDI-32|Mohan|Add| Docs for openmrs profile

mohan-13 committed:
BDI-32|Mohan|Add| Docs for Bahmni UI development

mohan-13 committed:
BDI-32|Mohan|Add| Docs for Bahmni Proxy Service

mohan-13 committed:
BDI-32|Mohan|Add| Docs for default URL and credentials for OpenMRS and Bahmni EMR

mohan-13 committed:
BDI-32|Mohan|Add| Docs for bahmni web env configurations

mohan-13 committed:
BDI-32|Mohan|Refactor| Docs for openelis URL to point to proxy URL

mohan-13 committed:
BDI-32|Mohan|Fix| Typo in README

gsluthra committed:
Merge pull request #67 from Bahmni/BDI-32

BDI-32 | Refactor | Docs for OpenMRS and Bahmni EMR Dockerisation

gsluthra committed:
Merge pull request #66 from Bahmni/BDI-27

BDI-27 | Add | Dockerisation of Proxy Service for Bahmni

mohan-13 committed:
BDI-33|Mohan|Fix| Missing profile configuration for openmrsdb service

mohan-13 committed:
BDI-29|Mohan|Refactor| Update build script to package default_config

mohan-13 committed:
BDI-29|Mohan|Add| COPY command in Dockerfile to include openelis config from default_config

mohan-13 committed:
BDI-29|Mohan|Refactor| OpenELIS start script for linking openelis config from default_config

mohan-13 committed:
BDI-29|Mohan|Refactor| OpenELIS liquibase XML to pick up from config folder

mohan-13 committed:
BDI-29|Mohan|Refactor| Docs for OpenELIS development operation

mohan-13 committed:
BDI-35|Mohan|Add| ADR for publishing snapshot images

mohan-13 committed:
BDI-35|Mohan|Refactor| Bahmni lab docker publish script to push SNAPSHOT images

mohan-13 committed:
BDI-35|Mohan|Refactor| Bahmni erp docker publish script to push SNAPSHOT images

mohan-13 committed:
BDI-35|Mohan|Refactor| Bahmni erp connect docker publish script to push SNAPSHOT images

gsluthra committed:
Merge pull request #68 from Bahmni/BDI-33

BDI-33 | Fix | Missing profile configuration for openmrsdb service

gsluthra committed:
Merge pull request #69 from Bahmni/BDI-29

BDI-29 | Add | Include default-config along with OpenELIS image

gsluthra committed:
Merge pull request #70 from Bahmni/BDI-35

BDI-35 | Add | Publishing SNAPSHOT docker images from Github Actions

mohan-13 committed:
BDI-29|Mohan|Fix| Directory creation issue in GH Actions

gsluthra committed:
Merge pull request #71 from Bahmni/BDI-29

BDI-29 | Fix | Directory creation issue in GH Actions

mohan-13 committed:
BDI-36|Mohan|Add| Docker publish script for bahmni openmrs

mohan-13 committed:
BDI-36|Mohan|Add| Docker publish script for bahmni emr ui

mohan-13 committed:
BDI-36|Mohan|Add| Docker publish script for bahmni proxy

mohan-13 committed:
BDI-36|Mohan|Add| README for bahmni openmrs docker scripts

mohan-13 committed:
BDI-36|Mohan|Add| README for bahmni emr ui docker scripts

gsluthra committed:
Merge pull request #72 from Bahmni/BDI-36

BDI-36 | Add | Docker publish scripts for Bahmni EMR Services

mohan-13 committed:
BDI-37|Mohan|Add| ADR for setting db tags to demo databases

mohan-13 committed:
BDI-38|Mohan|Refactor| Update image versions to current build for MVP

gsluthra committed:
Merge pull request #73 from Bahmni/BDI-38

BDI-38 | Refactor | Updating image tags to newer builds for MVP

mohan-13 committed:
Merge branch 'master' into BDI-37

mohan-13 committed:
BDI-37|Mohan|Refactor| Database image tags to be demo as default config

mohan-13 committed:
BDI-37|Mohan|Refactor| Update ADR status as Accepted

mohan-13 committed:
BDI-39|Mohan|Add| Volume mount for openmrs db data

gsluthra committed:
Merge pull request #75 from Bahmni/BDI-39

BDI-39 | Add | Volume mount for OpenMRS DB data

gsluthra committed:
Merge pull request #74 from Bahmni/BDI-37

BDi-37 | Refactor | Set demo database as default image tags

mohan-13 committed:
BAH-1266|Mohan|Refactor| Moved ADR to its individual repository

gsluthra committed:
Merge pull request #76 from Bahmni/BAH-1266

BAH-1266| Refactor | Moved ADR to its individual repository

mohan-13 committed:
BAH-1289|Mohan|Add| Prerequisites section to README

gsluthra committed:
Merge pull request #77 from Bahmni/BAH-1289

BAH-1289 | Add | Prerequisites section to README

mohan-13 committed:
BAH-1298|Mohan|Fix| Removed duplicated code

gsluthra committed:
Merge pull request #78 from Bahmni/BAH-1298

BAH-1298 | Fix | Removed duplicated code

mohan-13 committed:
BAH-1304|Mohan|Add| Steps to remove induilt fhir OMOD

gsluthra committed:
BAH-1310|Gurpreet|Updated README to explain steps for setting Odoo failed events to 1. And also added some clarification for default Bahmni app start regarding demo db and .env file

gsluthra committed:
BAH-1310|Gurpreet|Minor typo fix

mohan-13 committed:
Merge pull request #80 from Bahmni/BAH-1310

BAH-1310|Gurpreet|Updated README to explain steps to fix Odoo sync issue

mohan-13 committed:
BAH-1314|Mohan|Refactor| Bahmni-web developemnt steps to allow hot loading

gsluthra committed:
Merge pull request #79 from Bahmni/BAH-1304

BAH-1304 | Add | Steps to remove induilt FHIR OMOD

mohan-13 committed:
BAH-1314|Mohan|Refactor| Remove unwanted trailing slash

mohan-13 committed:
Merge pull request #81 from Bahmni/BAH-1314

BAH-1314| Mohan | Refactor | Bahmni-web development steps

Commit:
BAH-1275: Changes for adding open ssl certificate

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

Commit:
BAH-1275: Implementing encryption for certificate and key file.

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

Commit:
BAH-1275: Adding the encrypted certificate and key

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

Commit:
BAH-1275: Updating the readme for updation of SSL certificates

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

swatigogia2020 committed:
Update README.md

umair-fayaz committed:
BAH-1275 | Removed git-crypt, Created SSL configuration as part of image ceration

Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

N0-man committed:
Merge pull request #83 from Bahmni/BAH-1275

BAH-1275: Changes for adding open ssl certificate

umair-fayaz committed:
BAH-1276 | Configure client side logging

Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

Commit:
Merge branch 'master' into BAH-1276

deepthi-mantena committed:
BAH-1345 - [Deepthi,Rohit] - Upgrades log4j xml property files as per the latest version

deepthi-mantena committed:
BAH-1345 - [Deepthi,Rohit] -Modifies logger name for openmrs

deepthi-mantena committed:
BAH-1345 | Deepthi,Rohit | Migrates log4j conf to log4j2

umair-fayaz committed:
BAH-1276 | Added dependencies for client side logging, updated README

Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

umair-fayaz committed:
Resolve conflicts

umair-fayaz committed:
BAH-1276 | Removing the volume mount for client side logging, building it as part of proxy image

Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

N0-man committed:
Merge pull request #88 from Bahmni/BAH-1276

Bah 1276- Implemented Client-side logging

umair-fayaz committed:
BAH-1393 | Removed v0.94 from CI

Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

swatigogia2020 committed:
BAH-1275: SSL configuration for mod_cache and TLS/SSL , re-route configurations (#87)

* BAH-1338 | Added security configurations

Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

* BAH-1339 | Added mod_cache SSL configuration

Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

* BAH-1339 | Created Alias configurations

Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

* BAH-1275 | Sharing volumes between containers

Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

* BAH-1275 | Added maintenance configuration and standard environment variables cconfiguration (#86)

Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

Co-authored-by: Umair Fayaz <umair.fayaz@thoughtworks.com>
Co-authored-by: swatigogia2020 <swati.gogia@thoughtworks.com>

* BAH-1275: Made modification for mod_cache

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

* BAH-1275: Removed named volumes from bahmni-web container

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

* BAH-1275: Removed named volumes

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

Co-authored-by: Umair Fayaz <umair.fayaz@thoughtworks.com>
Co-authored-by: Umair Fayaz <59157924+umair-fayaz@users.noreply.github.com>
Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

umair-fayaz committed:
BAH-1393 | Changed required minimum memory to 8GB

Co-authored-by: Abinaya U <abinaya.u@thoughtworks.com>

mohan-13 committed:
BAH-1410 | added. configuration for implementer-interface container

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1410 | added. dependencies for implementer-interface

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1410 | added. development setup config for implementer-interface

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1410 | added. docs for implementer-interface

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1410 | added. httpd redirect rule for /home path

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
fixed: caching on api endpoints and redirect to https

mohan-13 committed:
Merge pull request #94 from Bahmni/cache-fix

FIX: Caching on API endpoints

mohan-13 committed:
Refactor | Updated proxy image version to add https support

mohan-13 committed:
Merge pull request #95 from Bahmni/proxy-version-update

Refactor | Updated proxy image version to add https support

mohan-13 committed:
Merge branch 'master' into BAH-1410

N0-man committed:
Merge pull request #89 from Bahmni/BAH-1345

BAH-1345 - [Deepthi,Rohit] - Upgrades log4j xml property files as per…

umair-fayaz committed:
Merge pull request #90 from Bahmni/BAH-1393

BAH-1393 | Removed v0.94 from CI

mohan-13 committed:
BAH-1416 | refactor. moved proxy docker scripts into bahmni-proxy subdirectory

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1416 | refactor. updated path references in bahmni-proxy build

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1416 | added. github actions to build proxy image

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1416 | added. workflow file path to trigger paths

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1416 | fixed. path mistach for checkout action

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1416 | added. environment variable for bahmni-version

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1416 | added. publish step to proxy image build

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

mohan-13 committed:
BAH-1416 | refactor. removed feature branch from trigger branches

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

SooryaKumaranC-tw committed:
BAH-1416 | Refactor naming

Co-authored-by: Arjun G <arjun.g@thoughtworks.com>

Arjun-Go committed:
BAH-1416 | build and upload embedded Tomcat Server artifact and trigger dependent workflows

Co-authored-by: Soorya Kumaran C <sooryakumaran.c@thoughtworks.com>

Arjun-Go committed:
BAH-1416 | add java version

Co-authored-by: Soorya Kumaran C <sooryakumaran.c@thoughtworks.com>

Arjun-Go committed:
BAH-1416 | refactor naming and removed testing files

Co-authored-by: Soorya Kumaran C <sooryakumaran.c@thoughtworks.com>

mohan-13 committed:
BAH-1416 | refactor. updated name of proxy build workflow

N0-man committed:
Merge pull request #96 from Bahmni/BAH-1416

BAH-1416 | Rearranging of docker resources and GHA for proxy image

mohan-13 committed:
refactor. proxy build workflow to docker build&push action

Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>

rohit-yawalkar committed:
BAH-1348 - | Rohit, Deepthi | - Upgrades Openmrs version to 2.1.6, at… (#97)

* BAH-1348 - | Rohit, Deepthi | - Upgrades Openmrs version to 2.1.6, atomfeed client to 1.10.0, bahmniRelease version to 0.94

* BAH-1345 - | Rohit, Deepthi | - Upgrades atomfeed version to 1.10.1 in gradle properties

N0-man committed:
Merge pull request #98 from Bahmni/gha-refactor

Refactor. proxy build workflow to use docker build&push action

rohit-yawalkar committed:
BAH-1315 - | Rohit, Deepthi | - Upgrades log4j as per we are getting from openmrs.war (#99)

N0-man committed:
initial K8s setup for Bahmni

umair-fayaz committed:
Revert "initial K8s setup for Bahmni -> Moving to feature branch"

This reverts commit 841a2d1e8cf7c573140e89d68eb17ea640105779.

SooryaKumaranC-tw committed:
BAH-1421 | Add trigger for bahmni-reports

mohan-13 committed:
BAH-1410 | refactor. updated image tag to latest

mohan-13 committed:
refactor. updated image tags to `latest` as default

mohan-13 committed:
added. common troubleshooting to docker README

mohan-13 committed:
Merge pull request #101 from Bahmni/imageTag-refactor

Refactor. Updating Image Tags to latest

mohan-13 committed:
Merge branch 'master' into BAH-1410

mohan-13 committed:
BAH-1410 | fixed. implementer-interface proxy path

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

rohit-yawalkar committed:
BAH-1315 - | Rohit, Deepthi | - Adds filePattern in log4j files (#102)

N0-man committed:
added config for bahmni lab lite

mohan-13 committed:
Merge branch 'master' into BAH-1410

SanoferSameera committed:
BAH 1563 | To generate trusted SSL certificate (#104)

* BAH-1563 | updated. proxy dockerfile to install certbot for certificate generation

* BAH-1563 | updated. readme with steps to generate trusted ssl certificate

* BAH-1563 | updated. readme with steps to generate trusted ssl certificate

SooryaKumaranC-tw committed:
BAH 1421 | Docker configuration for bahmni-reports (#100)

* BAH-1421 | added. configuration for bahmni-reports docker

* BAH-1421 | added. httpd proxy config for bahmni-reports docker

* BAH-1421 | refactor. extracted reports image as env variable

* BAH-1421 | Update readme for bahmni-reports

* (fix) bahmni-report image tag

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>
Co-authored-by: Abinaya U <abinaya.u@thoughtworks.com>
Co-authored-by: Arjun G <arjun.g@thoughtworks.com>
Co-authored-by: Kavitha S <kavitha.s@thoughtworks.com>
Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>
Co-authored-by: _n0man <nouman.memon@icloud.com>

vchrombie committed:
BAH-1553 | Add crater configurations to bahmni-docker

This commit updates bahmni-package configurations by
adding Crater services to the Bahmni docker-compose
solution along with a 'crater' profile.

mohan-13 committed:
BAH-1769 | refactor. openmrsdb setup from openmrs container

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

mohan-13 committed:
BAH-1769 | refactor. set openmrsdb image to mysql5.6 as default

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

mohan-13 committed:
BAH-1769 | fix. table formatting

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

mohan-13 committed:
BAH-1769 | refactor. update readme with database auto load variables

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

mohan-13 committed:
Merge branch 'master' into BAH-1410

mohan-13 committed:
Merge pull request #93 from Bahmni/BAH-1410

BAH-1410 |  Docker Configuration for implementer-interface

mohan-13 committed:
Merge branch 'master' into BAH-1769

sivareddyp committed:
Update bahmni-proxy.conf

sivareddyp committed:
Merge pull request #109 from Bahmni/lab-lite-issue-fix

Bahmni-Lab Issue - Update bahmni-proxy.conf

mohan-13 committed:
Merge pull request #108 from Bahmni/BAH-1769

BAH-1769 | Refactor. OpenMRS Container to create Database

kavitha-sundararajan committed:
BAH-1794 | added volume mounting for bahmni_config reports

kavitha-sundararajan committed:
BAH-1794 | refactored bahmni reports path to access from bahmni-config path

N0-man committed:
use root user in openmrs docker compose

kalyan90 committed:
BAH-1650 Bahmni docker setup on Macbook with M1 chip (#105)

* Bahmni docker setup on Macbook with M1 chip

* Update - Mutex posixsem for bahmni proxy

mohan-13 committed:
fix. python version in proxy image

mohan-13 committed:
BAH-1794 | refactor. docs for custom reports

umair-fayaz committed:
BAH-1816 | Added tags for disabling seeaerch engine crawling

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
Merge pull request #111 from Bahmni/BAH-1816

BAH-1816 | Added tags for disabling search engine crawling

binduak committed:
Bindu, Rohit | BAH-1872 | upgrade openmrs version to 2.1.7

binduak committed:
Bindu | BAH-1872 | fix permission issue with elis folder in uploaded-files (#112)

mohan-13 committed:
BAH-1809 | Refactor. remove use of db root credentials in openmrs container (#113)

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

mohan-13 committed:
Add. Workflow Dispatch to embedded tomcat build

abinaya-u committed:
BAH-1903 | Support Authorized access to patient documents (#115)

* BAH-1903 | add. patient-document service in docker-compose

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

* update. README

* refactor. updated README for patient-documents service

mohan-13 committed:
BAH-2000 | Add configuration for OpenMRS debug

mohan-13 committed:
BAH-2000 | Disable OpenMRS debugging by default

mohan-13 committed:
Merge pull request #117 from Bahmni/BAH-2000

BAH-2000 | Add configurations to enable debugging for OpenMRS

goyaldivij committed:
BAH-2003 | Enabling centralised logging for docker containers

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

mohan-13 committed:
BAH-2003 | Use YAML anchor for log config

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-2003 | Add proxy configurations for Grafana

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

goyaldivij committed:
BAH-2003 | Configuration to choose logging

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

mohan-13 committed:
BAH-2003 | Refactor use env variables

- Add Grafana volume mount for data
- Remove Grafana port publishing

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-2003 | Update README

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

vchrombie committed:
Update configurations based on PR review comments

mohan-13 committed:
Merge pull request #119 from Bahmni/BAH-2003

BAH-2003 | Loki Stack configuration for docker-compose

mohan-13 committed:
BAH-2054 | Add dockerisation files for dcm4chee,oviyam2

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

goyaldivij committed:
BAH-2054 | Added conf files for DCM4CHEE

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

stavtnameh committed:
BAH-1986| Changes for dockerization of Appointments-v2 (#118)

* BAH-1986| testing appointments-v2

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

* BAH-1986| testing appointments-v2

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

* BAH-1986| Appointments-v2 Dockerized

* BAH-1986 | Updated Volume Mount for appointment service

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

* BAH-1986 | Remove Appointments-V2 from url

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

* BAH-1986 | Update README for appointments docker setup


Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>
Co-authored-by: Divij Goyal <divij.g@beehyv.com>
Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

mohan-13 committed:
BAH-1422 | Add Bahmni PACS services to docker-compose.yml

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-1422 | Add proxy rules for dcm4chee,oviyam2

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-2054 | Refactor. ORM2DCM config file path.

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-2054 | Remove services from docker-compose.yml

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-1422 | Add persistent volume for dcm4chee archive

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-2054 | Refactor dcm4chee database initialisation

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-1422 | Refactor Use single postgres server for dcm4chee, pacs-integration

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

vchrombie committed:
Fix typos in the README.md and some wrong configurations

mohan-13 committed:
BAH-1422 | Add configuartion variables for PACS services

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

goyaldivij committed:
BAH-2054 | Added workflow file for dcm4chee docker build and publish

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

mohan-13 committed:
BAH-2054 | Remove downloaded artifacts from image after build

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-2054 | Remove feature branch from build workflow

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-2054 | Refactor. DCM4CHEE docker Startup Script

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

stavtnameh committed:
BAH-1938|Added crater-atomfeed configurations

stavtnameh committed:
BAH-1938|Added database configuration

mohan-13 committed:
BAH-2054 | Refactor bahmni-pacs Dockerfile

mohan-13 committed:
BAH-1422 | Add ports of DCM4CHEE

stavtnameh committed:
BAH-1938|Refactored docker-compose and .env

stavtnameh committed:
BAH-1938| Refactored secrets

mohan-13 committed:
BAH-1422 | Added missing env variable for pacs-simulator

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-1422 | Update README for Bahmni PACS Setup

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-1422 | Fix. Typos in README

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

abinaya-u committed:
BAH-2099 |  Add Unauthorized page in proxy

Co-authored-by: Arjun G <arjun.g@thoughtworks.com>
Co-authored-by: Siva Reddy <sivareddy.pathuri@thoughtworks.com>

mohan-13 committed:
Fix. Table formatting in bahmni-docker README

abinaya-u committed:
BAH-2099 | refactor. updated the title and copy text

Co-authored-by: Arjun G <arjun.g@thoughtworks.com>
Co-authored-by: Siva Reddy <sivareddy.pathuri@thoughtworks.com>

stavtnameh committed:
BAH-2139|first addition to compose and env

stavtnameh committed:
BAH-2139| Added proxy rules for atomfeed-console

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-2139|Added atomfeed-console to bahmni-package

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-2139| Added configs for openerp and openelis

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

umair-fayaz committed:
Merge with master

goyaldivij committed:
BAH-2163 | Added Volume Mount for clinical forms

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

stavtnameh committed:
BAH-2139| Added config for PACS Integration

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-1938|Refactored env variables in template

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-2139|Added triger for atomfeed console
Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

umair-fayaz committed:
BAH-2139 | Refactored | Removed redundant variable PACS_INTEGRATION_DB_HOST from .env

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2139 | Refactored | Removed exposed port 9080 for atomfeed-console

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2139 | Refactored | Renamed OpenERP appname to Odoo

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2139 | Added README for atomfeed-console

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2139 | Updated README for Docker

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2139 | Removed build atomfeed-console job

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2139 | Changed branch and organization name

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2139 | Fix | Typo resources

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2139 | Refactor | Removed chaining operator &&

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
Merge pull request #124 from Bahmni/BAH-2139

BAH-2139| Dockerized atomfeed-console

amankrayush-1 committed:
BAH-1904 | Add. rule to pass the request to vakyansh proxy

Co-authored-by: Priyanka Ajmera <pajmera@thoughtworks.com>

stavtnameh committed:
BAH-2193| Added trivy-scanner workflow

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-2193|Added trivy-scanner

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-2193|Added trivy-scanner

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-2193|Added trivy-scanner

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-2193|Added trivy-scanner

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-2193|Added paths to be ignored

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-2035|Fixed image storage problem

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>
Co-authored-by: Tuhin <tuhin.s@beehyv.com>

umair-fayaz committed:
BAH-2193 | Refactor | Added Trivy Scanner shell script

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2193 | Configured conditions for Trivy scan

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2193 | Refactored | Trivy scan to run from shell script

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2193 | Added empty space

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

umair-fayaz committed:
BAH-2193 | Refactored script

Co-authored-by: stavtnameh <hemanth.v@beehyv.com>

stavtnameh committed:
BAH-2193| changed trivy_scan bash script to consider the special paths

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
Delete something.sh

amankrayush-1 committed:
update readme file

Co-authored-by: Priyanka Ajmera <pajmera@thoughtworks.com>

mohan-13 committed:
BAH-2356 | Refactor. core project to bahmni-embedded-tomcat

stavtnameh committed:
BAH-2366| Refactor workflow to add homebrew to path and install trivy outside script

Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
BAH-2366| Refactored trivy setup steps into single step

Co-authored-by: goyaldivij <divij.g@beehyv.com>
Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>
Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>

stavtnameh committed:
Merge pull request #132 from Bahmni/BAH-2366

BAH-2366| Refactor workflow to add homebrew

goyaldivij committed:
BAH-2332 | Moved DCM4CHEE docker resources to pacs-integration repo (#129)

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

umair-fayaz committed:
BAH-2366 | Remove Trivy Brew based Installation

umair-fayaz committed:
BAH-2366 | Move Trivy installation steps to shell script

umair-fayaz committed:
Merge pull request #134 from Bahmni/BAH-2366

BAH-2366 | Remove Trivy Brew based Installation

stavtnameh committed:
BAH-1938|Added config for crater-atomfeed and crater-atomfeed-db (#130)

* BAH-1938|Added config for crater-atomfeed and crater-atomfeed-db

* BAH-1938|Added crater_url as env variable

* BAH-1938|Changed crater url in crater-atomfeed configs

* BAH-1938 | Added crater-atomfeed configuration.

* BAH-1938 | Fix. Crater URL for crater atomfeed sync

* BAH-1938 | Refactor. Update credentials of crater atomfeed

Co-authored-by: Divij Goyal <divij.g@beehyv.com>
Co-authored-by: Umair Fayaz <omayrfayaz@gmail.com>
Co-authored-by: MOHANKUMAR T <mohan13081999@gmail.com>

mohan-13 committed:
BAH-2436 | Use corretto as base image for atomfeed-console (#135)

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

NoorShiuli committed:
BAH-1712 | Add docker-compose architectural diagram (#128)

* BAH-1712 | Add docker-compose architectural diagram script

* BAH-1712 | Update docker-compose.yml file

abinaya-u committed:
BAH-2284 | Refactor. Landing page to display apps based on configuraton (#137)

mohan-13 committed:
BAH-2294 | Adds bahmni-config as a service (#133)

* BAH-2294 | Add. Bahmni-config as a seperate service and share as volume

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

* BAH-2294 | Update README

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

mohan-13 committed:
BAH-2383 | Refactor. Moved docker-compose to bahmni-docker repository (#138)

- Artifacts has been moved to [bahmni-docker](https://github.com/Bahmni/bahmni-docker) repository

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

Co-authored-by: Divij Goyal <divij.g@beehyv.com>

deepthi-mantena committed:
Deepthi M|BAH-3977|Updated log4j2.xml file and removed log4j listener as its no longer required with log4j2. (#141)

rahu1ramesh committed:
BAH-3969 | Add. Bahmni Embedded Tomcat v10 (#140)

* [Rahul] | BAH-3969 | Add. Bahmni Embedded Tomcat v10

* [Rahul] | BAH-3969 | Add. Main Class Exceptions

mohan-13 committed:
Fix. Upgrade setup-java action

rahu1ramesh committed:
[Rahul] | BAH-3969 | Fix. Missing Build Path

rahu1ramesh committed:
[Rahul] | Add. Missing Jakarta Dependency In Tomcat v10

rahu1ramesh committed:
[Rahul] | BAH-3969 | Fix. Tomcat 10 Build Files

rahu1ramesh committed:
[Rahul] | BAH-3969 | Fix. Tomcat 10 Build Root Project Name

rahu1ramesh committed:
BAH-3969 | Add. logback-classic, javax.servlet, Gradle Wrapper Support For Tomcat 10 

* [Rahul] | BAH-3969 | Add. logback-classic As Dependency In Tomcat 10

* [Rahul] | BAH-3969 | Add. javax.servlet As Dependency In Tomcat 10

* [Rahul] | BAH-3969 | Add. Gradle Wrapper Support For Tomcat 10

rahu1ramesh committed:
BAH-3969 | Add. Gradle Wrapper Jar

rahu1ramesh committed:
BAH-3944 | Fix. Markdown Violations In README.md

rahu1ramesh committed:
[Rahul] | BAH-3944 | Add. Bump Checkout Version

rahu1ramesh committed:
[Rahul] | BAH-4097 | Add. Tag Based Trigger And Version Script To Build Workflow (#143)

rahu1ramesh committed:
[Rahul] | BAH-4097 | Add. appVersionFile Path To atomfeed-console Workflow (#145)

rahu1ramesh committed:
[Rahul] | BAH-4116 | Add. Bump App Version 1.0.0 -> 1.1.0 (#146)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Bahmni/bahmni-package?pullRequest=3) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/Bahmni/bahmni-package?pullRequest=3) before we can accept your contribution.<br/>**9** out of **10** committers have signed the CLA.<br/><br/>:white_check_mark: umair-fayaz<br/>:white_check_mark: stavtnameh<br/>:white_check_mark: mohan-13<br/>:white_check_mark: NoorShiuli<br/>:white_check_mark: goyaldivij<br/>:white_check_mark: abinaya-u<br/>:white_check_mark: rahu1ramesh<br/>:white_check_mark: deepthi-mantena<br/>:white_check_mark: vivekbiswas-tw<br/>:x: amankrayush-1<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Bahmni/bahmni-package?pullRequest=3) it.</sub>

vivekbiswas-tw committed:
Initialize connector for tomcat 10 (#147)

- Since tomcat version 9.x, the connector has to be initialized manually to start listening to HTTP requests. If not initialized, the tomcat server would start, but will not listen to any HTTP requests.

Reference: https://stackoverflow.com/questions/71383171/simple-embedded-tomcat-10-example

mohan-13 committed:
BAH-4548 | Refactor. Remove trivy script and use shared script from infra-utils (#149)
